otter/integration/lib/autoscale.py

```diff
@@ -181,20 +197,23 @@ def get_scaling_group_state(self, rcs):
             order.  If not found, the result code will be 404, and the state
             will be None.
         """
+        if success_codes is None:
```

Comment:
Non-blocking :: this can be expressed more simply as `success_codes = success_codes or [200, 404]`